content/assets/blog/phoenix-case-study/index.md

```diff
@@ -0,0 +1,150 @@
+---
+title: "Phoenix: An end-to-end AI powered Social Media analysis platform powering peacebuilding
+efforts"
+date: "2025-02-10T10:16:57Z"
+description: "Phoenix is an open-source AI platform developed by Build Up and datavaluepeople to
+aid peacebuilding through social media analysis. Launched in January 2025, it addresses challenges
+like large data processing, limited technical expertise, and under-resourced-language
+classification. Key features include a user-friendly interface, secure authentication, data
+scraping, and multilingual classification. Phoenix has already supported hundreds peacebuilders,
+processed terabytes of data, and gained recognition at the Paris Peace AI Forum. A public-good
+platform, making advanced AI tools accessible for conflict analysis of digital spaces. Phoenix
+exemplifies how collaboration and ethical AI can drive global peacebuilding efforts."
+author: benjamin
+linkedin_link:
+hackernews_link:
+---
+In a world where social media deeply shapes public opinion, data-driven insights can play a pivotal
+role in preventing conflict and fostering peace. That’s exactly what [Build
+Up](https://howtobuildup.org/phoenix/), a nonprofit dedicated to supporting peacebuilders—sought to
+accomplish through the Project Phoenix. Developed in partnership with datavaluepeople, Phoenix is
+an open-source (code repository [here](https://gitlab.com/howtobuildup/phoenix)), end-to-end AI
+powered platform designed to help peacebuilders and mediators allowing them to analyze social media
+discourses, informing effective strategies for conflict prevention and resolution. Since January
+2025 phoenix has been open to the public and you can apply for a Phoenix account
```

Comment:
Do a case-sensitive search for "phoenix" and make them all "Phoenix".

Reply:
f41bb75